generate_markdown_from_doxygen_xml.py

```diff
@@ -239,7 +241,20 @@ def markdown_any_tag(aTag, html=False,para=True,consume=False):
     return tag_text
 
 
-
+def seealso_from_tag(aTag):
```

Comment:
FYI, the default position of seealso tags was not the optimal. Using this function we make them part of the class so we can render them wherever we need.